front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -55,15 +54,15 @@ export class CreateF3XStep1Component implements OnInit, OnDestroy {
     [F3xReportCodes.Q3, 'OCTOBER 15 QUARTERLY REPORT(Q3)'],
     [F3xReportCodes.YE, 'JANUARY 31 YEAR-END (YE)'],
     [F3xReportCodes.TER, 'TERMINATION REPORT (TER)'],
-    [F3xReportCodes.MY, 'JULY 31 MID-YEAR REPORT (NON-ELECTION YEAR ONLY) (MY)'],
-    [F3xReportCodes.TwelveG, '12-DAY PRE-ELECTION GENERAL (12G)'],
-    [F3xReportCodes.TwelveP, '12-DAY PRE-ELECTION PRIMARY (12P)'],
-    [F3xReportCodes.TwelveR, '12-DAY PRE-ELECTION RUNOFF (12R)'],
-    [F3xReportCodes.TwelveS, '12-DAY PRE-ELECTION SPECIAL (12S)'],
-    [F3xReportCodes.TwelveC, '12-DAY PRE-ELECTION CONVENTION (12C)'],
-    [F3xReportCodes.ThirtyG, '30-DAY POST-ELECTION GENERAL (30G)'],
-    [F3xReportCodes.ThirtyR, '30-DAY POST-ELECTION RUNOFF (30R)'],
-    [F3xReportCodes.ThirtyS, '30-DAY POST-ELECTION SPECIAL (30S)'],
+    [F3xReportCodes.MY, 'JULY 31 MID-YEAR REPORT (MY)'],
```

Comment:
It's actually a good thing sonar cloud brought it up!  We forgot to delete `F3xReportCodeLabels` in `f3x-summary.model.ts` when we replaced it in #331.  If you delete that label list (the one in the `f3x-summary.model.ts` not this one), it will eliminate the duplication warning 😄